packages/react-dom-bindings/src/client/ReactDOMComponent.js

```diff
@@ -711,6 +712,7 @@ function setProp(
     case 'disableRemotePlayback':
     case 'formNoValidate':
     case 'hidden':
+    case enableNewBooleanProps ? 'inert' : 'formNoValidate':
```

Comment:
This is quite hacky but allows for a minimal diff as far as I can tell. We want the case to be just `case 'inert'` when `enableNewBooleanProps` is enabled and ideally the case to disappear without `enableNewBooleanProps` which we get indirectly by resuing a value we already covered earlier. I didn't choose `hidden` because that might move in the future when we add `hidden="until-found"`.

It's quite lazy so open to refactoring this if you feel uncomfortable shipping it like this.

Reply:
Alternate would be: [`133867b` (#24730)](https://github.com/facebook/react/pull/24730/commits/133867bd0f1eb9c77b43ba6e3a14276362c3cb39)

I'm not sure how to add it behind a flag without runtime impact.